processor/src/test/java/org/mapstruct/ap/test/abstractclass/AbstractDto.java

```diff
@@ -22,6 +22,10 @@
  * @author Andreas Gudian
  */
 public class AbstractDto implements Identifiable {
+
+    //CHECKSTYLE:OFF
```

Comment:
Does CS complain about public fields? I'd say we should modify the rule set in this case.